src/mongocxx/test/CMakeLists.txt

```diff
@@ -173,6 +173,13 @@ add_executable(test_unified_format_spec
     ${spec_test_common}
 )
 
+add_executable(test_versioned_api
+    ${PROJECT_SOURCE_DIR}/test_util/client_helpers.cpp
+    ${THIRD_PARTY_SOURCE_DIR}/catch/main.cpp
+    versioned_api.cpp
```

Comment:
This may need to be the full path to `versioned_api.cpp`.

The Evergreen patch build attached to this PR is reporting errors in cmake configuration. Example: https://spruce.mongodb.com/task/cxx_driver_arm_ubuntu1804_50_compile_and_test_with_shared_libs_patch_c2932becd7b69dfa15bfc6a8cda6038dfc33a600_610af93d7742ae1ea40b23f7_21_08_04_20_32_00/logs?execution=0

Reply:
Looks like the file needed to be added to the dist list later down in the file.